incubator/binding-mqtt.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/streams/network/publish.one.message.subscribe.as.one.packet/client.rpt

```diff
@@ -0,0 +1,65 @@
+#
```

Comment:
I think packet may not be the correct terminology here.
In MQTT spec, each of `publish` and `subscribe` are defined as specific "packets", so they cannot be combined in the same "packet".
Perhaps we can say "unfragmented" or similar instead as it seems to relate to the delivery of `DATA` frame payload in the stream, not at the MQTT layer per se, agree?